SmartCredentials_aOS/storage/README.md

```diff
@@ -0,0 +1,114 @@
+
+# Usage of the Smart Credentials Storage API
+
+Smart Credentials Storage Module is used for persisting items in a secure and reliable manner.
+
+### Add build.gradle dependencies
+```
+implementation("de.telekom.smartcredentials:core:$sc_version")
+implementation("de.telekom.smartcredentials:security:$sc_version")
+implementation("de.telekom.smartcredentials:storage:$sc_version")
+````
+
+### Modules initialization
+
+The Smart Credentials module initialization is recommended to be made in the application class.
+
+**Core Module**
+``` 
+SmartCredentialsConfiguration coreConfig = new SmartCredentialsConfiguration.Builder(context, userId)
+                .setLogger(new Logger())
+                .setAppAlias(alias)
+                .build();
+CoreApi coreApi = SmartCredentialsCoreFactory.initialize(coreConfig);
+````
+
+**Security module**
+``` 
+SecurityApi securityApi = SmartCredentialsSecurityFactory.initSmartCredentialsSecurityModule(this, coreApi);  
+````
+
+**Storage module**
+``` 
+SmartCredentialsStorageFactory.initSmartCredentialsStorageModule(this, coreApi, securityApi);
+````
+
+### Add item to storage
+
+**Step 1:** Create item context
+
+The item context defines the item as being sensitive or non-sensitive. The sensitive repository use Android's ```SharedPreferences``` as the storage location and the non-sensitive repository use a database as the storage location. 
+The sensitive repository is recommended for small items and operations on it does not need to be performed on a background thread, while the sensitive repository is recommended for large items and operations on it must be made on a background thread.
```

Comment:
..while the **non-sensitive** repository is recommended for large items and operations on it must be made on a background thread.

Reply:
whooops ... fixed now